ee/codegen/src/utils/nodes.ts

```diff
@@ -28,3 +28,7 @@ export function getNodeLabel(nodeData: WorkflowNode): string {
       return nodeData.data.label;
   }
 }
+
+export function isUnaryOperator(operator: string): boolean {
+  return operator === "null" || operator === "notNull";
```

Comment:
[oos] we have a few more descriptors that will be considered unary (undefined is two). But since we don't have serialization or UI support for them either, let's be sure to track for now